CI/batch/docker/Dockerfile.gpu

```diff
@@ -1,4 +1,4 @@
-FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/autogluon-training:0.6.2-gpu-py38-cu113-ubuntu20.04
+FROM 763104351884.dkr.ecr.us-east-1.amazonaws.com/autogluon-inference:0.7.0-gpu-py39-cu117-ubuntu20.04
```

Comment:
why change from training to inference? Also this docker is used by our CI so should be built and push to our ECR before merging in

Reply:
Good catch, will switch back to training

Reply:
Talked with @yinweisu offline, agreed the update of image is not 0.8 relevant and I will revert this file change for now to unblock 0.8 release